File: python/raindrops/raindrops.py
def convert(number):
    '''Given a number, will convert it to a nonsense phrase.'''
    answer = ''
    if not number % 3: answer += "Pling"
    if not number % 5: answer += "Plang"
    if not number % 7: answer += "Plong"

    return answer or str(number)

File: r/raindrops/raindrops.R
raindrops <- function(number) {
  rain = ""
  if (number %% 3 == 0){
    paste(rain, "Pling", sep="")}
  else if (number %% 5 == 0) {
    paste(rain, "Plang", sep="")}
  else if (number %% 7 == 0) {
    paste(rain, "Plong", sep="")
  }
  else 
    as.character(number)
  #"Pling"
}
